tempesta_fw/t/functional/regression/test_shutdown.py

```diff
@@ -40,38 +43,41 @@ def init(self):
         defconfig = 'cache 0;\n'
         self.tempesta.config.set_defconfig(defconfig)
         self.create_servers()
+        for server in self.servers:
+            server.start("Server")
         self.configure_tempesta()
-        self.tempesta.start()
+        self.tempesta.start("Tempesta")
```

Comment:
May be it's better to move `obj` from `Stateful.start()` to `Stateful.__init__()`? Then  the object name will be available while `Stateful` class instance is alive.

Reply:
In my variant we can for debug purposes change names in the place